backend/src/main/java/ch/puzzle/okr/service/business/ObjectiveBusinessService.java

```diff
@@ -26,15 +28,21 @@ public class ObjectiveBusinessService implements BusinessServiceInterface<Long,
     private final ObjectiveValidationService validator;
     private final KeyResultBusinessService keyResultBusinessService;
     private final CompletedBusinessService completedBusinessService;
+    private final ActionPersistenceService actionPersistenceService;
+    private final ActionBusinessService actionBusinessService;
 
     public ObjectiveBusinessService(@Lazy KeyResultBusinessService keyResultBusinessService,
                                     ObjectiveValidationService validator,
                                     ObjectivePersistenceService objectivePersistenceService,
-                                    CompletedBusinessService completedBusinessService) {
+                                    CompletedBusinessService completedBusinessService,
+                                    ActionPersistenceService actionPersistenceService,
+                                    ActionBusinessService actionBusinessService) {
```

Comment:
action business service should be enough there should be no need to use action persisstence service here